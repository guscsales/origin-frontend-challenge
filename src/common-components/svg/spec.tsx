import * as React from 'react';
import { create } from 'react-test-renderer';
import Svg from '.';
import svgTest from '../../assets/svg/icons/logo.svg';

describe('Svg', () => {
  describe('#render', () => {
    it('should match a snapshot', () => {
      const wrapper = create(
        <Svg src={svgTest} width="10px" height="10px" />
      ).toJSON();

      expect(wrapper).toMatchSnapshot();
    });
  });
});
