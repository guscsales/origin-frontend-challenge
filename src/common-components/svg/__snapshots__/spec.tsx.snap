// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Svg #render should match a snapshot 1`] = `
<svg
  height="10px"
  width="10px"
>
  <use />
</svg>
`;
